src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/displayDescriptionData_List.cs

```diff
@@ -201,7 +201,7 @@ internal override bool CompatibleWithOldPowerShell()
     public sealed class ListControlEntry
     {
         /// <summary>List of items in the entry</summary>
-        public List<ListControlEntryItem> Items { get; internal set; }
+        public List<ListControlEntryItem> Items { get; set; }
```

Comment:
The comment here should be updated per the standard format (`Gets or sets ...`)

Reply:
There are a handful of other cases where we're making setters public; we should do the same for all of them.